semanticdb-javac/src/main/java/com/sourcegraph/semanticdb_javac/SemanticdbVisitor.java

```diff
@@ -502,6 +507,17 @@ private int semanticdbSymbolInfoProperties(Symbol sym) {
     return properties;
   }
 
+  private List<String> semanticdbOverrides(Symbol sym) {
```

Comment:
I think you can do like this
```suggestion
  private List<String> semanticdbOverrides(Symbol sym) {
    return new ArrayList<>(javacTypes.getOverriddenMethods(sym));
```

Reply:
Maybe even just return the set directly
```java
private Iterable<String> semanticdbOverrides(Symbol sym) {
  return javacTypes.getOverriddenMethods(sym);
}
```

Reply:
`getOverriddenMethods` returns a `Set<Symbol.MethodSymbol>`, hence the need to get the semanticdb symbol string for each